src/main/resources/js/condec.context.menu.js

```diff
@@ -91,6 +91,10 @@
 
 		document.getElementById("condec-context-menu-sentence").style.zIndex = 9998;
 		document.querySelector("#condec-context-menu-sentence").setAttribute('aria-hidden', 'false');
+		
+		document.getElementById("condec-context-menu-sentence-create-item").onclick = function() {
```

Comment:
@Jcl15 , I added this because we also want to add new elements from sentence objects.